randovania/games/samus_returns/logic_database/Area 5 Tower Interior.txt

```diff
@@ -458,7 +433,7 @@ Extra - asset_id: collision_camera_007
 > Above Pickup; Heals? False
   * Layers: default
   > Door to Interior Teleporter (Bottom)
-      Morph Ball
+      Grapple Beam and Morph Ball
```

Comment:
this feels like we should have another event for being able to move the shuffler.

Reply:
I don't think so, since to get in, you either move the Grapple block or clip. The Grapple requirement here is more for the standard path to prevent the clip method from using it.

Reply:
event would be nice for the other door case as well.

Reply:
I'm pretty sure that woudn't help (and the reason I didn't add an event), because the shuffler resets its position on scenario reload.

Reply:
oh